ghcide/src/Development/IDE/Core/OfInterest.hs

```diff
@@ -32,24 +34,40 @@ import qualified Data.ByteString                          as BS
 import           Data.Maybe                               (catMaybes)
 import           Development.IDE.Core.ProgressReporting
 import           Development.IDE.Core.RuleTypes
-import           Development.IDE.Core.Shake
+import           Development.IDE.Core.Shake               hiding (Log,
+                                                           logToPriority)
+import qualified Development.IDE.Core.Shake               as Shake
 import           Development.IDE.Plugin.Completions.Types
 import           Development.IDE.Types.Exports
 import           Development.IDE.Types.Location
 import           Development.IDE.Types.Logger
+import qualified Development.IDE.Types.Logger             as Logger
 import           Development.IDE.Types.Options            (IdeTesting (..))
 import qualified Language.LSP.Server                      as LSP
 import qualified Language.LSP.Types                       as LSP
+import           Prettyprinter                            (Pretty (pretty))
+
+data Log = LogShake Shake.Log
```

Comment:
I think that would be OK but I also feel like if we needed to add more logging it would be less confusing for there to be a template already existing that could be added to.

Reply:
Fair enough.